scripts/minio.py

```diff
@@ -0,0 +1,158 @@
+import json
+import os
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from django.conf import settings
+
+from grandchallenge.core.storage import public_s3_storage
+
+
+def run():
+    """Sets up the permissions on the minio buckets"""
+    print("🔐 Setting up minio 🔐")
+
+    if not settings.DEBUG:
+        raise RuntimeError(
+            "Skipping this command, server is not in DEBUG mode."
+        )
+
+    _setup_public_storage()
+    _setup_components_storage()
+
+    print("✨ Minio set up ✨")
+
+
+def _setup_public_storage():
+    """
+    Add anonymous read only to public S3 storage.
+
+    Only used in development. In production, set a similar policy on the S3 bucket.
+    """
+    host_alias = "local"
+    policy_name = "public_read"
+    policy = {
+        "Version": "2012-10-17",
```

Comment:
Would it make sense to move this to a setting as well?

Reply:
No IMO, this really as defines the schema of the policy, not the version of the policy itself https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html